core-services/prow/02_config/openshift-assisted/assisted-events-scrap/_pluginconfig.yaml

```diff
@@ -0,0 +1,9 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift-assisted/assisted-events-scrap
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift-assisted/assisted-events-scrap
+  review_acts_as_lgtm: true
```

Comment:
I think it needs also jira plugin, just like we did for assisted-installer-deployment:
https://github.com/openshift/release/blob/9668b6bb2b2d5c91d46601150bf8c2592529db71/core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_pluginconfig.yaml#L14